django/webscraper/web_scraper.py

```diff
@@ -70,7 +89,8 @@ def parse_pet(href):
 
 
 def __timed_get(href):
-    """Performs get request with preset timeout"""
+    """Performs get request with preset timeout, with a delay"""
+    time.sleep(3)  # added in order not to get banned from the server

```

Comment:
Consider making delay optional / configurable  by for example file constant at the file top